source/features/add-co-authored-by.js

```diff
@@ -0,0 +1,53 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import {getUsername, escapeForGql} from '../libs/utils';
+import onNewComments from '../libs/on-new-comments';
+
+async function addCoAuthoredBy() {
+	const usernameElements = select.all('.js-discussion .author:not(.rgh-fullname):not([href*="/apps/"])');
```

Comment:
As said by Sindre, instead of the comments it should just get the commits, something that can probably be done with a single `pullRequest -> commits -> users -> email` GraphQL query

Reply:
Good idea on the GraphQL query, I hadn't had a reason to really play with the GitHub GraphQL API before now. 🙂